webserver/src/main/java/io/kestra/webserver/services/BasicAuthService.java

```diff
@@ -75,6 +76,12 @@ public void save(String uid, BasicAuthConfiguration basicAuthConfiguration) {
             throw new IllegalArgumentException("No password set for Basic Authentication. Please provide a password.");
         }
 
+        if (basicAuthConfiguration.getUsername().length() > EMAIL_PASSWORD_MAX_LEN ||
+            basicAuthConfiguration.password.length() > EMAIL_PASSWORD_MAX_LEN) {
+            throw new IllegalArgumentException("The length of email or password should not exceed 256.");
```

Comment:
Add the word `characters` at the end of this error message, to be `The length of email or password should not exceed 256 characters.`